CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 26.5.0
```

Comment:
You won't know if there will be a release before this is merged, so the convention is to leave the version number out or put it under a `Unreleased` heading.